src/sdk/resources/sites.ts

```diff
@@ -1,9 +1,17 @@
 import { IHttpClient } from '../http';
-import { EditableResource } from './resource';
+import { EditableResource, IDownloadFileProvider } from './resource';
 import { RemoteSite } from '../models';
 
-export class SitesResource extends EditableResource<RemoteSite> {
+export class SitesResource extends EditableResource<RemoteSite> implements IDownloadFileProvider {
 	constructor(httpClient: IHttpClient) {
 		super(httpClient, 'site');
 	}
+
+	getHttpClient(): any {
```

Comment:
This doesn't need to be exposed here. The `SkySyncClient` already exposes the http client.